app/models/manageiq/providers/openstack/inventory/collector/cloud_manager.rb

```diff
@@ -53,7 +53,7 @@ def host_aggregates
   def images
     return @images if @images.any?
     @images = if openstack_admin?
-                image_service.handled_list(:images, {}, true).all
+                image_service.images_with_pagination_loop
               else
                 image_service.handled_list(:images)
```

Comment:
Do we need pagination loop also in case that is_admin: false?

Reply:
When is_admin => false, `handled_list` automatically handles pagination so this should be unnecessary.

Reply:
Thanks, merging :+1: